IPython/lib/display.py

```diff
@@ -327,7 +327,7 @@ def __init__(self,
             prefix to be prepended to all files to form a working link [default:
             '']
         result_html_prefix : str
-            text to append to beginning to link [default: none]
+            text to append to beginning to link [default: '']
```

Comment:
Changed this one for consistency with `url_prefix` docstring update.

First time looking at IPython code, so don't know if the use of `none` here is idiomatic or not. If not, I can drop this commit from the PR.